README.adoc

```diff
@@ -0,0 +1,33 @@
+# Gravitee Risk Assessment Api
```

Comment:
If the README is migrated to `adoc`, shouldn't we delete the one written in markdown?